src/resolvers/mosaic.resolver.ts

```diff
@@ -43,14 +43,23 @@ export class MosaicIdAliasResolver implements Resolver {
      * @param {string} altText - Alternative text.
      * @returns {MosaicId | NamespaceId}
      */
-    resolve(options: ProfileOptions, secondSource?: Profile, altText?: string): any {
+    resolve(options: ProfileOptions, secondSource?: Profile, altText?: string): MosaicId | NamespaceId {
         const resolution = OptionsResolver(options,
             'mosaicId',
             () =>  undefined,
             altText ? altText : 'Enter the mosaic id or alias: ').trim();
         new MosaicIdAliasValidator().validate(resolution);
         return MosaicService.getMosaicId(resolution);
     }
+
+    optionalResolve(options: any, altKey?: string, defaultValue?: string): MosaicId | NamespaceId {
```

Comment:
Documentation is missing. I'm referring to the comment placed before the function describing each parameter passed and the general definition of what it does.